docs/README.md

```diff
@@ -0,0 +1,44 @@
+# Docs
+Meta information about this project's documentation.
+
+Table of contents:
+- [Building the docs](#Building)
+- [Writing docstrings](#Docstrings)
+- [Writing a changelog](#Changelog)
+
+
+## Building
+To build the docs, you can use the following task:
+```shell
+poetry run task docs
+```
+
+The output will be in the [`/docs/build`](.) directory.
+
+Additionally, there are two helper tasks: `apidoc` and `builddoc`.
+`apidoc` is responsible for calling autodoc and generating docs from docstrings.
+`builddoc` generates the HTML site, and should be called after apidoc.
+
+Neither of these two tasks needs to be manually called, as the `docs` task calls both.
+
+
+## Docstrings
+To have your code probably documented, you need to do a couple of things:
```

Comment:
Oh wait is this for it to appear in Sphinx?

Maybe mention that like "For your code to show up in the automatically generated documentation..".

Reply:
I'm really not sure why this clarification is necessary. I'm going to reword it I guess since I'm going to be editing this line, but I mean, this is a general guide on writing your docstrings...

Reply:
e465278e